### ISSUE TITLE
New page placement: it would be nice to have possibility to add new page into some nodes

### ISSUE THREAD

**Issue by daivabrazukaite** [devbridge/BetterCMS#252]
Create new page -> page placement window will be opened. Sitemap edit mode allows to add page to sitemap in different positions, but this feature is not available in this window. Here you are able to add page only once. I think that it should be similar like in sitemap and after page is added, it should not disappear from left side area. If this will be implemented, also page deletion/edit node should be available (i.e. all functionality that is available in "site settings -> site map" window main part + adding new page into sitemap)
